lib/ddtrace/contrib/action_cable/integration.rb

```diff
@@ -0,0 +1,40 @@
+require 'ddtrace/contrib/integration'
+require 'ddtrace/contrib/action_cable/configuration/settings'
+require 'ddtrace/contrib/action_cable/patcher'
+
+module Datadog
+  module Contrib
+    module ActionCable
+      # Description of ActionCable integration
+      class Integration
+        include Contrib::Integration
+
+        register_as :action_cable, auto_patch: false
+
+        def self.version
+          Gem.loaded_specs['rails'] && Gem.loaded_specs['rails'].version
```

Comment:
Is ActionCable packaged as a gem or have a gem version? If so we should probably reflect that here, instead of the Rails version.